MicrosoftFluentUI.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
   s.name             = 'MicrosoftFluentUI'
-  s.version          = '0.4.9'
+  s.version          = '0.5.0'
```

Comment:
This is pretty much the only change from the main-tokens branch, right?

Reply:
Also, should this be changed to 5.0 now, or should there be a change just after this to bump this and the rest of the version bump info?

Reply:
Yep @rdeassis mentioned that too. I'll do that separately.